Gems/ScriptCanvas/Code/Include/ScriptCanvas/Libraries/Math/MathExpression.ScriptCanvasGrammar.xml

```diff
@@ -17,7 +17,7 @@
                   Description="The resulting string."
                   Type="double"
                   DisplayGroup="ExpressionDisplayGroup"
-                  IsInput="False"
-                  IsOutput="True" />
+                  IsInput="True"
```

Comment:
Seems like you could just add the RestrictedTypeContract here
`Contracts="RestrictedTypeContract{{ToAZType(ScriptCanvas::Data::Type::Number()), ToAZType(ScriptCanvas::Data::Type::Vector3())}}"`